Michael-Masas/bootstrap.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+sudo yum install -y yum-utils;
+sudo yum-config-manager \
+    --add-repo \
+    https://download.docker.com/linux/centos/docker-ce.repo;
+sudo yum install -y docker-ce docker-ce-cli containerd.io;
```

Comment:
You have have used a bit different package list than what Liroy have used. I would suggest that you both check what is the minimal amount of packages required.

Reply:
Those packages are from official Docker Docs , i don't see a reason to change it.